exes/button/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+add_executable(button src/main.cpp)
+
+find_path(AZMQ_INCLUDE_DIRS "azmq/actor.hpp")
```

Comment:
azmq is not explicitly used, but sml is used, so maybe change to sml

Reply:
copy paste error... fixed